packages/shared/src/components/modals/SlackIntegrationModal.tsx

```diff
@@ -0,0 +1,351 @@
+          >
+            <Typography
+              tag={TypographyTag.Span}
+              bold
+              color={TypographyColor.Primary}
+            >
+              {source.name}
+            </Typography>{' '}
+            <Typography
+              tag={TypographyTag.Span}
+              color={TypographyColor.Tertiary}
+            >
+              @{source.handle}
+            </Typography>
+          </Typography>
+        </div>
+        <Typography
+          tag={TypographyTag.H3}
+          type={TypographyType.Title2}
+          color={TypographyColor.Primary}
+          bold
+        >
+          Configure Integration
+        </Typography>
+        {!!slackIntegrations?.length && (
+          <Dropdown
+            placeholder="Select workspace"
+            shouldIndicateSelected
+            buttonSize={ButtonSize.Medium}
+            iconOnly={false}
+            key="feed"
+            selectedIndex={selectedIntegrationIndex}
+            renderItem={(value, index) => {
+              const currentIntegration = slackIntegrations[index];
+              const ItemIcon =
+                currentIntegration.id === 'new' ? PlusIcon : SlackIcon;
+
+              return (
+                <span className="flex gap-2 typo-callout">
+                  <ItemIcon />
+                  {slackIntegrations[index].name}
+                </span>
+              );
+            }}
+            icon={<SlackIcon />}
+            options={slackIntegrations?.map((item) => item.name)}
+            onChange={(value, index) => {
+              const currentIntegration = slackIntegrations[index];
+
+              if (currentIntegration.id === 'new') {
+                onConnectNew();
+
+                return;
+              }
+
+              setState((current) => {
+                return {
+                  ...current,
+                  userIntegration: slackIntegrations[index],
+                  channelId: undefined,
+                };
+              });
+            }}
+            scrollable
+          />
+        )}
+        <div>
+          <Typography
+            className="mb-2 flex items-center"
+            type={TypographyType.Body}
+            color={TypographyColor.Tertiary}
+          >
+            Select channels
+            {!!channels?.length && (
+              <Typography
+                className="ml-1 flex min-h-6 min-w-6 items-center justify-center rounded-full bg-surface-float p-0.5"
+                tag={TypographyTag.Span}
+                type={TypographyType.Callout}
+                color={TypographyColor.Tertiary}
+                bold={false}
+              >
+                {channels.length}
+              </Typography>
+            )}
+          </Typography>
+          {!!channels?.length && (
+            <Dropdown
+              placeholder="Select channel"
+              shouldIndicateSelected
+              buttonSize={ButtonSize.Medium}
+              iconOnly={false}
+              key="feed"
```

Comment:
feed is ok, not sure it's used already?

Reply:
it is per component, but yeah, better to update, its weird